src/status_im/contexts/wallet/bridge/bridge_to/view.cljs

```diff
@@ -19,12 +19,14 @@
     (let [network                     (rf/sub [:wallet/network-details-by-chain-id chain-id])
           currency                    (rf/sub [:profile/currency])
           currency-symbol             (rf/sub [:profile/currency-symbol])
+          prices-per-token            (rf/sub [:wallet/prices-per-token])
           balance                     (utils/calculate-total-token-balance token [chain-id])
-          crypto-value                (utils/get-standard-crypto-format token balance)
+          crypto-value                (utils/get-standard-crypto-format token balance prices-per-token)
           fiat-value                  (utils/calculate-token-fiat-value
-                                       {:currency currency
-                                        :balance  balance
-                                        :token    token})
+                                       {:currency         currency
+                                        :balance          balance
+                                        :token            token
+                                        :prices-per-token prices-per-token})
```

Comment:
Could we have subs for `utils/get-standard-crypto-format` and `utils/calculate-token-fiat-value`, since in a few places it seems to be directly using values from other subs?

Reply:
These are utility functions and they receive different parameters in different contexts (except for `prices-per-token`), so I believe they aren't very suitable for being a subscription.

But I understand what is the problem that becomes more obvious. We have a piece of information (`prices-per-token`) that is not needed by UI itself, but have to be passed to utility functions. So the UI here is just a middleman that operates unknown data for no reason. And we have more cases like this. For example passing `:profile/currency` from ui to utility functions to conversion functions.

I believe that subscription is not a best way to solve this. That data belongs to lower layer (we can think of it as of business-logic), so it should be accessed on lower layer. For example we can create a `money/conversion api` for ui that contains same functions but they receive less arguments. And calls then redirected to utils functions but that additional arguments like `currency`, `prices-per-token`, etc.. taken from database and UI don't know about them.

Reply:
agreed, best would be to figure it out during the devx meeting